app.js

```diff
@@ -1,13 +1,22 @@
 const express = require('express');
 
+const bodyParser = require('body-parser');
+
 const ItemHandler = require('./ItemHandler');
 
 const itemHandler = new ItemHandler();
 
 const app = express();
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
 app.set('view engine', 'pug');
 
-app.get('/', (req, res) => {
+app.get('/item', (req, res) => {
+  res.render('index', { message: 'Hello World!', listOfItems: itemHandler.getList() });
+});
+
+app.post('/item', (req, res) => {
+  itemHandler.createNewItem(req.body.itemName);
```

Comment:
we should identify the Accept header and return the JSON or the HTML

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

Reply:
Resolve in commit: 8ead7106b4f4227e480fb68cd9ced11b67e73dc7